common/source/docs/common-teraranger-one-rangefinder.rst

```diff
@@ -72,3 +72,9 @@ left bottom windows where quick values are displayed. Then choose
    :target: ../_images/TeraRangerOne_MissionPlannerEnableFlightData.jpg
 
    Mission Planner: Setting to display rangefinder data on the Flight Datascreen
+
+.. warning::
+
+When the Pixhawk boots, the TeraRanger One needs to already be powered on, or at 
```

Comment:
Hi @lfrodrigues 

Sphinx is quite sensitive to indentation - it is basically very like Python programming. That is why you will see in my original response I indented each line by 4 spaces. Can you do the same thing?

So:

```
.. warning::
  
    When the Pixhawk boots, t
    ....
```